src/source/WasmGenerator.scala

```diff
@@ -230,30 +230,34 @@ class WasmGenerator(spec: Spec) extends Generator(spec) {
     val helper = helperClass(ident)
     writeHppFileGeneric(spec.wasmOutFolder.get, helperNamespace(), wasmFilenameStyle)(ident.name, origin, refs.hpp, Nil, (w => {
       w.w(s"struct $helper: ::djinni::WasmEnum<$cls>").bracedSemi{
-        w.wl("static void staticInitialize();");
+        if (!spec.wasmOmitConstants) {
```

Comment:
I can't comment on the exact line, but no need for the mutex include on L228 without static init

Reply:
The refs are put in cpp files.  They are no longer generated so it doesn't matter.